docs/lib/graphqlapi/fragments/android/authz/21_oidc.md

```diff
@@ -0,0 +1,26 @@
+Add the following code to your app:
+
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+ApiAuthProviders authProviders = ApiAuthProviders.builder()
+    .oidcAuthProvider(() -> {
+        // TODO: Amplify.Auth.fetchAuthSession
```

Comment:
@raphkim  could you help out with this code snippet? 

by default, we will use the accessToken for CUP auth type, however we've seen customers requiring to use the ID Token (https://github.com/aws-amplify/amplify-ios/issues/780). This example shows how to use the Cognito as the OIDC provider, and get the ID Token from the auth session

Reply:
Added code snippet in the following commit: https://github.com/aws-amplify/docs/pull/2211/commits/395310cade970c56aea9c925ad1cdaf4704236f7